packages/datetime/src/timePicker.tsx

```diff
@@ -252,10 +274,30 @@ export class TimePicker extends React.Component<ITimePickerProps, ITimePickerSta
         this.updateTime(loopTime(newTime, unit), unit);
     }
 
+    private isTimeInRange(date: Date) {
+        const format = "H:mm:ss:SSS";
+        const time = moment(this.getDateOnlyWithTime(date), format);
+        const minTime = moment(this.getDateOnlyWithTime(this.props.minTime), format);
+        const maxTime = moment(this.getDateOnlyWithTime(this.props.maxTime), format);
+
+        const isTimeAfterMinTime = time.isSameOrAfter(minTime);
+        const isTimeBeforeMaxTime = time.isSameOrBefore(maxTime);
```

Comment:
is it possible to do this _without_ relying on `moment`? just using the regular `DateUtils` we already have?